src/Phpbb/TranslationValidator/Validator/FileListValidator.php

```diff
@@ -201,6 +201,11 @@ public function validate()
 					}
 					$this->output->addMessage($level, 'Found additional file', $origin_file);
 				}
+
+				if (substr($origin_file, -13 ) === 'site_logo.gif' || substr($origin_file, -14 ) === '/site_logo.gif')
```

Comment:
I think we need to keep the second case only in case the language pack author decided to have a similarly named logo file, i.e. he_site_logo.gif, which I think would be ok.